locale/en_US/emails.po

```diff
@@ -312,3 +312,15 @@ msgstr ""
 "<p>If you have any questions, please contact me from your <a href=\"{$authorSubmissionUrl}\">submission dashboard</a>.</p>"
 "<p>Kind regards,</p>"
 "<p>{$signature}</p>"
+
+msgid "emails.reviewResendRequest.subject"
+msgstr "Requesting your review again for {$contextName}"
+
+msgid "emails.reviewResendRequest.body"
+msgstr ""
+"<p>Dear {$recipientName},</p>"
+"<p>Recently, you declined our request to review a submission, {$submission Title}, for {$contextName}. I'm writing to see if you are able to conduct the review after all.</p>"
```

Comment:
The typo was in my suggestion, but there's an extra space in `{$submission Title}`.

Reply:
fixed.